portal.go

```diff
@@ -1966,3 +1967,42 @@ func (portal *Portal) UpdateInfo(source *User, meta *discordgo.Channel) *discord
 	}
 	return meta
 }
+
+func (portal *Portal) ForwardBackfill(source *User) error {
+	portal.log.Debugln("Checking for missing messages to fill")
+	lastMessage := portal.bridge.DB.Message.GetLast(portal.Key)
+	if lastMessage == nil {
+		portal.log.Debugln("No last message in portal, can't forward backfill")
+		return nil
+	}
+
+	// Get up to 100 messages at a time until everything is fetched
+	for {
+		messages, err := source.Session.ChannelMessages(portal.Key.ChannelID, 100, "", lastMessage.DiscordID, "")
```

Comment:
I think the discord web app gets 50 at a time

```suggestion
		messages, err := source.Session.ChannelMessages(portal.Key.ChannelID, 50, "", lastMessage.DiscordID, "")
```